src/features/users/authorSlice.ts

```diff
@@ -0,0 +1,18 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { User } from '../../types/User';
+
+const initialState = null as User | null;
```

Comment:
The initial state is typed as 'User | null', which is good for type safety. However, consider initializing it explicitly to 'null' for better readability and to avoid potential issues with type inference in the future.